src/main/java/nextstep/courses/domain/SessionDuration.java

```diff
@@ -10,16 +11,16 @@ public class SessionDuration {
     private static final String TYPE_3 = "시작일은 현재시간의 최소 7일 이후에 등록해야됩니다.";
     private static final int MIN_DURATION = 7;
 
-    private LocalDate startedAt;
-    private LocalDate endedAt;
+    private LocalDateTime startedAt;
+    private LocalDateTime endedAt;
 
-    public SessionDuration(LocalDate startedAt, LocalDate endedAt) {
+    public SessionDuration(LocalDateTime startedAt, LocalDateTime endedAt) {
         checkCourseDuration(startedAt, endedAt);
         this.startedAt = startedAt;
         this.endedAt = endedAt;
     }
 
-    void checkCourseDuration(LocalDate startedAt, LocalDate endedAt) {
+    void checkCourseDuration(LocalDateTime startedAt, LocalDateTime endedAt) {
```

Comment:
```suggestion
    private void checkCourseDuration(LocalDateTime startedAt, LocalDateTime endedAt) {
```

private으로 구현해도 되지 않을까?